src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -698,7 +729,51 @@ void ov::npuw::CompiledModel::implement_properties() {
           }}},
         {ov::loaded_from_cache.name(), {ov::PropertyMutability::RO, [&](const ::intel_npu::Config&) {
                                             return m_loaded_from_cache;
-                                        }}}};
+                                        }}},
+        // OV Public Hints
+        // =========
+        {ov::hint::performance_mode.name(),
+         {ov::PropertyMutability::RO,
+          [&](const ::intel_npu::Config& config) {
+              GET_PUBLIC_PROP(ov::hint::performance_mode);
+          }}},
+        {ov::hint::execution_mode.name(),
+         {ov::PropertyMutability::RO,
+          [&](const ::intel_npu::Config& config) {
+              GET_PUBLIC_PROP(ov::hint::execution_mode);
+          }}},
+        {ov::hint::num_requests.name(),
+         {ov::PropertyMutability::RO,
+          [&](const ::intel_npu::Config& config) {
+              GET_PUBLIC_PROP(ov::hint::num_requests);
+          }}},
+        {ov::hint::inference_precision.name(),
+         {ov::PropertyMutability::RO,
+          [&](const ::intel_npu::Config& config) {
+              GET_PUBLIC_PROP(ov::hint::inference_precision);
+          }}},
+        {ov::hint::enable_cpu_pinning.name(),
+         {ov::PropertyMutability::RO,
+          [&](const ::intel_npu::Config& config) {
+              GET_PUBLIC_PROP(ov::hint::enable_cpu_pinning);
+          }}},
+        {ov::hint::model_priority.name(),
+         {ov::PropertyMutability::RO, [&](const ::intel_npu::Config& config) {
+              int npu_submodel_idx = -1;
+              for (size_t i = 0; i < m_compiled_submodels.size(); ++i) {
+                  auto comp_model_desc = m_compiled_submodels[i];
+                  if (!comp_model_desc.compiled_model)
+                      continue;
+                  if (submodel_device(i) == "NPU") {
+                      npu_submodel_idx = i;
+                      break;
+                  }
+              }
+              return npu_submodel_idx > -1 ? m_compiled_submodels[npu_submodel_idx].compiled_model->get_property(
+                                                 ov::hint::model_priority.name())
+                                           : ov::hint::Priority::DEFAULT;
```

Comment:
Why no `GET_PUBLIC_PROP` here?

Reply:
Unfortunately, this property isn't supported by `CPU`, that is why I have hardcoded it in case `NPU` submodel isn't found.

Reply:
Please, put a comment here